types/migrations/20220205042039_init.sql

```diff
@@ -3,6 +3,8 @@
 
 create extension if not exists "uuid-ossp";
 
+create extension if not exists "ltree";
```

Comment:
Never ever edit database migrations that are already merged to main and applied on live deployments. We always have to make new migration files for new changes, because postgres skips completed migrations based on the file name and our migration process will fail if an existing migration is edited. If you navigate to the `types` folder, make a new migration with `sqlx migrate new migration_name`. It'll automatically prepend the date and create a .sql file for you.